.github/actions/reliable-pull-request/.gitignore

```diff
@@ -0,0 +1,4 @@
+.git
```

Comment:
No need to push this, since those dirs only live on your local machine

Reply:
I agree as the scope (use) of this file is not relevant to the import of this GitHub Action.